src/Microsoft.VisualStudio.Editors/Common/Utils.vb

```diff
@@ -1665,5 +1666,29 @@ Namespace Microsoft.VisualStudio.Editors.Common
 
         End Class
 #End Region
+
+        Public Class SerializationProvider
```

Comment:
>SerializationProvider [](http://example.com/codeflow?start=21&length=21)

How about a name like ObjectSerializer? I think the Provider suffix here is confusing because this class has only static methods and doesn't appear to "provide" any object as the provider pattern does, assuming Provider is even a pattern [https://blog.ploeh.dk/2011/04/27/Providerisnotapattern/](https://blog.ploeh.dk/2011/04/27/Providerisnotapattern/)

Reply:
At the time, I forgot our made-up 'provider' naming thing meant what the industry normally calls a factory. This is not a factory. However, it does *provide* serialization. But, I agree, `ObjectSerializer` would work just fine here.

Reply:
I've renamed it.